src/compiler/core.ts

```diff
@@ -2,8 +2,9 @@
 /// <reference path="performance.ts" />
 
 namespace ts {
+    export const versionMajorMinor = "2.5";
     /** The version of the TypeScript compiler release */
-    export const version = "2.5.0";
+    export const version = `${versionMajorMinor}.0`;

```

Comment:
versionMajorMinor was required in several places (not just tests), but previously we were parsing it out of `version`. Figured it would be better to construct `version` from `versionMajorMinor` than the other way around.